Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -669,6 +670,119 @@ Multisite also allows object stores to be independent and isolated from other ob
 
 For more information on multisite please read the [ceph multisite overview](ceph-object-multisite.md) for how to run it.
 
+## Object Multi-instance
+
+This section describes how to configure multiple `CephObjecStore` backed by the same storage pools.
```

Comment:
There are multiple places where the `t` is missing in the CR name, not just this one
```suggestion
This section describes how to configure multiple `CephObjectStore` backed by the same storage pools.
```